src/checkout_body/checkout_body.css

```diff
@@ -0,0 +1,207 @@
+.checkout_section {
+    padding: 86px 174px 44px 175px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.checkout_header {
+    font-size: 36px;
+    font-weight: 600;
+    letter-spacing: .5px;
+}
+
+.checkout_form {
+    padding-top: 36px;
+}
+
+.double_iputs_container {
+    display: flex;
+    gap: 31px;
+}
+
+.standart_chek_input_container {
+    display: flex;
+    flex-direction: column;
+    padding-top: 36px;
+}
+
+.double_iputs_item {
+    display: flex;
+    flex-direction: column;
+}
+
+.inputs_label {
+    font-size: 16px;
+    font-weight: 600;
+    padding-bottom: 22px;
+}
+
+.double_checkkout_input {
+    width: 211px;
+    height: 75px;
+}
+
+
+.standart_chek_input {
+    width: 453px;
+    height: 75px;
+}
+
+.check_input {
+    font-size: 16px;
+    font-weight: 400;
+    border-radius: 10px;
+    border: 1px solid #9F9F9F;
+    background: #FFF;
+}
+
+.checkout_selector {
+    color: #9F9F9F;
+    width: 453px;
+    height: 75px;
+    padding-left: 25px;
+    position: relative;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    background-image: url('/public/photos/64.svg');
```

Comment:
why is it `64.svg`? what does it even mean? name your files according to their meaning